build-system/babel-plugins/babel-plugin-mangle-object-values/index.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @fileoverview Mangles the values of an object expression.
+ * This is useful when defining large enums, as it allows us to use descriptive
+ * values during development, but use short mangled values during production.
+ */
+
+const {encode, indexCharset} = require('base62/lib/custom');
+
+const fnName = 'mangleObjectValues';
+
+module.exports = function (babel) {
+  const {types: t} = babel;
+
+  let charset;
+
+  /**
+   * @param {number} i
+   * @return {string}
+   */
+  function mangle(i) {
+    if (!charset) {
+      // Letters first since they allow unbracketed syntax when used as keys:
+      // (foo.a0 vs. foo['0a'])
+      charset = indexCharset(
```

Comment:
Optional: If there are only 10 (or fewer) enum values, we could optimize to a number instead. That'd save a few bytes.

Reply:
If we use numbers let's please exclude `0` that's falsy

Reply:
From my local output, using numbers from `0..99` seems to reduce the `amp-story` bundle by a further 75 B compressed. Cool!